docs/dev_database/common/segbits.rst

```diff
@@ -0,0 +1,115 @@
+=============
+segbits files
+=============
+
+The *segbits files* are generated for every FPGA :term:`tile <tile>` type.
```

Comment:
Need a description of "default' features, e.g. features that only require bits to be "unset".

Reply:
I've added appropriate information